scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -893,11 +892,10 @@ class FossId internal constructor(
             PackageProvider.RUBYGEMS -> PurlType.GEM
 
             else -> {
-                issues += FossId.createAndLogIssue(
-                    source = "FossId",
-                    message = "Cannot determine PURL type for url '$url' and provider '$provider'."
-                )
-                null
+                logger.warn {
+                    "Cannot determine PURL type for url '$url' and provider '$provider'. Falling back to 'generic'."
+                }
+                PurlType.GENERIC
```

Comment:
I propose

                PurlType.GENERIC.also {
                    logger.warn {
                        "Cannot determine PURL type for url '$url' and provider '$provider'. Falling back to '$it'."
                    }
                }


to not hard-code the string representation of `PurlType.GENERIC`.

Reply:
Sorry there has been a mix up and I fixed it in the wrong branch.

It has been pushed now:
https://github.com/oss-review-toolkit/ort/compare/5416fa09f20251e030b0cda9958537a90426d79b..0f39a48bd367abfc01442350cee78eae4088334d

Reply:
Wait, it's not there :(